mlflow/utils/uri.py

```diff
@@ -42,9 +42,15 @@ def get_db_info_from_uri(uri):
     """
     parsed_uri = urllib.parse.urlparse(uri)
     if parsed_uri.scheme == "databricks":
-        parsed_path = parsed_uri.path.lstrip('/') or None
-        parsed_profile = parsed_uri.netloc
-        return parsed_profile, parsed_path
+        profile_tokens = parsed_uri.netloc.split(':')
+        parsed_profile = profile_tokens[0]
+        if len(profile_tokens) == 1:
```

Comment:
There are shorter possible ways to write this in code (the `len(profile_tokens) == 1` case is actually not necessary, since `:.join(..)` will not include a `:` if the length of the join list is 1), but I think it may be better to just be explicit about what's happening.